xml/Global.xml

```diff
@@ -104,4 +104,17 @@
   </Panel>
 </VerticalLayout>
 
+<!-- Title Splash when starting a scenario -->
+<Text id='title_splash'
+  position='0 0 0'
+  fontSize="150"
+  font="font_teutonic-arkham"
+  outline="black"
+  outlineSize="3 -3"
+  showAnimation='FadeIn'
+  hideAnimation='FadeOut'
+  active='false'
+  animationDuration='2'>
```

Comment:
Is this big enough to handle even very long names?  Probably want to set this to wrap just in case

Reply:
It wraps by default. I tested a long name. It even wraps by word so you don't have words that are cut in half

Reply:
Ah, the TTS docs say the default is Overflow, I'd never tried it without setting it to wrap.  Thanks for adding it anyway, will be good for clarity.